megamek/src/megamek/client/ui/swing/calculationReport/CalculationReport.java

```diff
@@ -203,4 +203,21 @@ private String getRoundedResultString(String resultPrefix, double result) {
         return ((resultPrefix != null) ? resultPrefix : "")
                 + String.format(Locale.US, "%1$,.1f", result);
     }
+
+    /**
+     * Formats the given double with only the necessary digits and at most three digits. Uses the fixed Locale.US
+     * as the Java way of converting "" + value seems to use Locale.US by default as well.
+     */
+    static String fmt(double d) {
```

Comment:
Can we name this a little more descriptively? "fmt" doesn't really tell me anything about what the method does.

Reply:
Invited that change request with such a name :). There's these lines:
bvReport.addLine("Offensive Battle Rating:",
                    fmt(oBV) + " x " + fmt(speedFactor), "= " + fmt(oBV * speedFactor));
where the method is used three times. Making it "formatDoubleForReport" or "formatReportThreeDigits" or something like that would make the report calls very long and I guess rather less readable. I'm all for descriptive names but had hoped to keep this one short and let people mouse hover it to see what it does. (I couldn't think of a good way to have the report do the formatting by itself). Call it format instead?

Reply:
OK changed the name and made a few small corrections. I couldn't think of any name that was more descriptive and not ridiculously long.